### PR TITLE
fix: ovs docs formatting errors and AI docs typo

### LINKED ISSUE #4154: get-started-ovs-zh_CN Document format is messed up
### The Type of Document Issue

Wrong

### What's wrong with this document?

https://spidernet-io.github.io/spiderpool/v0.9/usage/install/underlay/get-started-ovs-zh_CN/#open-vswitch

img_v3_02fg_5a22bf59-6257-4529-b820-5886f15cb0bg

### PR DESCRIPTION
**Pull request by ty-dc**
# Thanks for contributing!

**Notice**:

* [x] unite test or E2E test
* [x] do not forget essential code comment and log
* [x] document for the PR
* [x] release note label
  "release/none"
  "release/bug"
  "release/feature"
* [x] read about  Contribution notice: <https://spidernet-io.github.io/spiderpool/latest/develop/contributing/>

**What issue(s) does this PR fix**:

Fixes #4154 

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by ty-dc** [comment]
image

File: docs/usage/install/ai/get-started-macvlan.md [hunk 1]
```diff
@@ -142,7 +142,7 @@ The network planning for the cluster is as follows:
 
 ## Install Spiderpool
 
-1. Use Helm to install Spiderpool and enable the SR-IOV component:
+1. Use Helm to install Spiderpool and enable the rdmaSharedDevicePlugin component:
```
Comment (windsonsea) [inline docs/usage/install/ai/get-started-macvlan.md:145]:
```suggestion
1. Use Helm to install Spiderpool and enable the rdmaSharedDevicePlugin:
```
`component` is not very necessary, and may cause confuse with Plugin.

File: docs/usage/install/underlay/get-started-ovs-zh_CN.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ Spiderpool 可用作 Underlay 网络场景下提供固定 IP 的一种解决方
 
 2. 在 /etc/netplan 目录下创建 12-br1.yaml 后，通过 `netplan apply` 生效。为确保在重启主机等场景下 br1 仍然可用，请检查 eth0 网卡是否也被 netplan 纳管。
 
-    ```yaml: 12-br1.yaml
+    ```yaml
```
Comment (windsonsea) [inline docs/usage/install/underlay/get-started-ovs-zh_CN.md:59]:
```suggestion
    ```yaml title="12-br1.yaml"
```
If you want to add a title, refer to the syntax:
https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#adding-a-title

Comment (ty-dc) [reply docs/usage/install/underlay/get-started-ovs-zh_CN.md]: done

File: docs/usage/install/ai/get-started-macvlan.md (hunk 1 above)
Comment (ty-dc) [reply docs/usage/install/ai/get-started-macvlan.md]: done

**Review by windsonsea — APPROVED** [review]

**Review by weizhoublue — APPROVED** [review]

**Pull request merged**